client/src/pages/chat/Chat.jsx

```diff
@@ -0,0 +1,113 @@
+import AddPrescription from "../../components/chatComponents/chatPrescriptions/addPrescription/AddPrescription.jsx";
+
+const DisplayType = Object.freeze({
+  DOCTOR: 1,
+  PATIENT: 2,
+});
+
+/* 
+
+    displayInfo:
+    For PATIENT : string ; 
+        eg "Common Cold"
+        {
+          userName: "Alanakov Bavonoichi",
+          unreadMessageCount: "5",
+          displayInfo: "Common Cold",
+        }
+    For DOCTOR : array ; 
+        eg ["Surgeries", "Anesthetics", "Valorant", "Punjabi"]
+        {
+          userName: "Alanakov Bavonoichi",
+          unreadMessageCount: "5",
+          displayInfo: ["Surgeries", "Anesthetics", "Valorant", "Punjabi"],
+        }
+*/
+
+function Chat() {
+  let chatID = 0;
+
+  const [closeSessionState, setCloseSessionState] = useState(false);
+  const [closeSessionDiagnosis, setCloseSessionDiagnosis] = useState("");
+  const [closeSessionComments, setCloseSessionComments] = useState("");
+  const [addPrescriptionState, setAddPrescriptionState] = useState(false);
+  const [displayType, setDisplayType] = useState(DisplayType.DOCTOR);
+  const fetchChatUsers = [
+    {
+      userName: "Alanakov Bavonoichi",
+      unreadMessageCount: "5",
+      displayInfo: ["Surgeries", "Anesthetics", "Valorant", "Punjabi"],
+    },
+  ];
+
+  return (
+    <>
+      <Navbar isFixed={true} />
+      <CloseSessionBox
+        closeSessionDiagnosisState={{
+          closeSessionDiagnosis: closeSessionDiagnosis,
+          setCloseSessionDiagnosis: setCloseSessionDiagnosis,
+        }}
+        closeSessionCommentsState={{
+          closeSessionComments: closeSessionComments,
+          setCloseSessionComments: setCloseSessionComments,
+        }}
+        closeSessionPassState={{
+          closeSessionState: closeSessionState,
+          setCloseSessionState: setCloseSessionState,
+        }}
+      />
+      <AddPrescription
+        AddPrescriptionState={{
+          addPrescriptionState: addPrescriptionState,
+          setAddPrescriptionState: setAddPrescriptionState,
+        }}
+      />
+      <div className="chat-container">
+        <div
+          className="chat-nav"
+          style={fetchChatUsers.length <= 0 ? { textAlign: "center" } : {}}
+        >
+          {fetchChatUsers.length <= 0 ? (
+            <span className="empty-chat-nav">No users found</span>
+          ) : (
+            fetchChatUsers.map((chatUsers) => {
+              return (
+                <SelectChat
+                  userName={chatUsers.userName}
+                  unreadMessageCount={chatUsers.unreadMessageCount}
+                  displayInfo={chatUsers.displayInfo}
+                  viewType={displayType}
+                  key={chatID++}
+                />
+              );
+            })
+          )}
+        </div>
+        <div className="chat-text-area">
+          <ChatHead userName={"Alanakov Banovoichi"} />
+          <DisplayMessages />
+          <TypingBar />
```

Comment:
dummy data

Reply:
It might be drastically changed in the future so I have just hardcoded the userName.